lib/reporting/drop_off_report.rb

```diff
@@ -82,7 +82,7 @@ def overview_table
         ['Report Timeframe', "#{time_range.begin} to #{time_range.end}"],
         # This needs to be Date.today so it works when run on the command line
         ['Report Generated', Date.today.to_s], # rubocop:disable Rails/Date
-        ['Issuer', issuers.join(', ')],
+        ['Issuer', [issuers].flatten.join(', ')],
```

Comment:
Isn't this change a noop if we're putting values into an array and then immediately flattening it back out?

```suggestion
        ['Issuer', issuers.join(', ')],
```

Reply:
Yes. This change allows `issuers` to either be a single item or an array or any array-like object. We've had some problems where people updating the configs aren't clear on which to use, so this accepts both

Reply:
Gotcha, the flattening of a new array jumped out to me as redundant on first glance, though I understand now how it could be used with singular or even nested values.

I might have expected more explicit test setups with the different forms we're expecting to support / avoid regressions on, though I _think_ I see them covered in the new tests.

Reply:
I can certainly make that more explicit

Reply:
https://github.com/18F/identity-idp/compare/19f07152a58aacd69e30013e457b14f4cd3669e1..aa987434c0de7ac00776a0616974eef3ae56d27c

Reply:
Thanks for adding 👍

Reply:
If we want to support array and single items, we can do
```suggestion
        ['Issuer', Array(issuers).join(', ')],
```